indexfile.html

```diff
@@ -0,0 +1,114 @@
+            <li class="s3">Codepen</li>
+            <li class="s4">Gitlab</li>
+            <li class="s5">Terminal</li>
+          </ul>
+        </div>
+</section><hr>
+<div class="socialicon">
+        <div class="link">
+          <img src="img/SocialMedia.png" alt="Social Media">
+        </div>
+        <div class="workicon">
+          <img src="img/arrowdown.png" alt="Arrow">
+          <h4>Work</h4>
+        </div>
+</div>
+<section class="work">
+  <h2>MY RECENT WORK</h2><hr>
+  <section class="w1">
+    <div class="imgplaceholder"></div>
+    <h3>Multi-Post Stories</h3>
+    <p class="w1p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <div class="prolang">
+    <p class="css">css</p>
+    <div class="vl"></div>
+    <p class="html">html</p>
+    <div class="vl"></div>
+    <p class="bs">bootstrap</p>
+    <div class="vl"></div>
+    <p class="ruby">Ruby</p>
+  </div>
+    <button type="button" name="See Project">See Project</button>
+  </section>
+  <section class="w2">
+    <div class="imgplaceholder"></div>
+    <h3>Multi-Post Stories</h3>
+    <p class="w2p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <div class="prolang">
+    <p class="css">css</p>
+    <div class="vl"></div>
+    <p class="html">html</p>
+    <div class="vl"></div>
+    <p class="bs">bootstrap</p>
+    <div class="vl"></div>
+    <p class="ruby">Ruby</p>
+  </div>
+    <button type="button" name="See Project">See Project</button>
+  </section>
+  <section class="w3">
+    <div class="imgplaceholder"></div>
+    <h3>Multi-Post Stories</h3>
+    <p class="w3p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <div class="prolang">
+    <p class="css">css</p>
+    <div class="vl"></div>
+    <p class="html">html</p>
+    <div class="vl"></div>
+    <p class="bs">bootstrap</p>
+    <div class="vl"></div>
+    <p class="ruby">Ruby</p>
+  </div>
+    <button type="button" name="See Project">See Project</button>
+  </section>
+  <section class="w4">
+    <div class="imgplaceholder"></div>
+    <h3>Multi-Post Stories</h3>
+    <p class="w4p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <div class="prolang">
+    <p class="css">css</p>
+    <div class="vl"></div>
+    <p class="html">html</p>
+    <div class="vl"></div>
+    <p class="bs">bootstrap</p>
+    <div class="vl"></div>
+    <p class="ruby">Ruby</p>
+  </div>
```

Comment:
- [ ] Please kindly wrap the list of project technologies (e.g., "JavasScript", "HTML" and "CSS")  in a `<ul>` instead of a `<div>`, and each language or framework should be wrapped in an `<li>` tag. Also remember to use proper indentations in your code, as it makes you code cleaner and easy to read.